pkg/minikube/tunnel/kic/ssh_tunnel.go

```diff
@@ -120,6 +139,31 @@ func (t *SSHTunnel) startConnection(svc v1.Service) {
 	}
 }
 
+func (t *SSHTunnel) startConnectionIngress(ingress v1_networking.Ingress) {
+	uniqName := sshConnUniqNameIngress(ingress)
+	existingSSHConn, ok := t.conns[uniqName]
+
+	if ok {
+		// if the svc still exist we remove the conn from the stopping list
+		delete(t.connsToStop, existingSSHConn.name)
+		return
+	}
+
+	resourcePorts := []int32{80}
```

Comment:
80, 443

Reply:
Hi,
Thanks for your attention. I add port 443 as well.

But you may know the certificate problem below

```bash
$ curl https://hello-world.info
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

```bash
curl -k https://hello-world.info
Hello, world!
Version: 1.0.0
Hostname: web-79d88c97d6-xl5gv
```

Reply:
Hi, 
Thanks for taking my comment into consideration
The way I tested

```shell
# start new minikube with ingress addon
minikube start --nodes=2 --addons=ingress

# create workload
kubectl create deployment web --replicas=2 \
   --image=gcr.io/google-samples/hello-app:1.0

# create service
kubectl expose deployment web \
   --type=NodePort --port=8080
```

generate self-singed certificates for hello-world.info using [this doc](https://github.com/iliadmitriev/openssl-scripts#usage)

```shell
# create tls secret
kubectl create secret tls hello-world-tls \
  --cert hello-world.info.pem \
  --key hello-world.info.key

# create ingress
kubectl create ingress hello-world-ingress \
  --rule=hello-world.info/=web:8080,tls=hello-world-tls
```

add `127.0.0.1 hello-world.info` to `/etc/hosts`

start tunnel
```shell
minikube tunnel
```
  
check
```shell
curl https://hello-world.info/
```

and this worked

Reply:
I'm glad to know it worked for you.
Besides, it tried your openssl-script and it worked great as well.